src/main/java/jpa/domain/LineStation.java

```diff
@@ -0,0 +1,46 @@
+package jpa.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class LineStation {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@ManyToOne
+	@JoinColumn(name = "line_id")
+	private Line line;
+
+	@ManyToOne
+	@JoinColumn(name = "station_id")
+	private Station station;
+
+	private int distance;
```

Comment:
이렇게 해도 요구사항은 충족되지만 아래 부분에 대해서도 한번 고민해보시면 좋을 것 같아요.

```
객체 지향 생활 체조 원칙
규칙 3: 모든 원시값과 문자열을 포장한다.
```

아래 키워드로 학습 및 검색 진행해보시면 도움이 될 것 같습니다.

`@Embeddable, @Embedded`

Reply:
이전 역과의 거리를 나타내는 값인데, 이전 역에 대한 정보가 추가되면 어떨까요?

현재는 이전 역과의 거리는 나타낼 수 있겠지만, 이전 역이 어딘지 알 수 없을 것 같아요

```suggestion
    @ManyToOne
    @JoinColumn(name = "pre_station_id")
    private Station preStation;
```